packages/components/src/Combobox/hooks/useComboboxActivatorAccessibility.ts

```diff
@@ -0,0 +1,16 @@
+import { useContext } from "react";
+import { ComboboxContext } from "../ComboboxProvider";
+import { COMBOBOX_MENU_ID } from "../constants";
+
+export function useComboboxActivatorAccessibility() {
+  const { label, open } = useContext(ComboboxContext);
+
+  return {
+    htmlAttributes: {
```

Comment:
specifying the html-ness of this and making it a distinct set because I had wanted to apply them to the `Combobox.Trigger` too, however that's a `Chip` and throwing these on it does nothing. it would want React props eg. `ariaControl`